microsoft/testsuites/network/common.py

```diff
@@ -169,18 +169,17 @@ def sriov_vf_connection_test(
 
         # check the connectivity between source and dest machine using ping
         for _ in range(max_retry_times):
-            cmd_result = source_node.execute(
-                f"ping -c 1 {dest_ip} -I {source_synthetic_nic}", sudo=True
+            ping = source_node.tools[Ping]
+            ping_result = ping.ping(
+                target=dest_ip, nic_name=source_synthetic_nic, count=1, sudo=True
             )
-            if cmd_result.exit_code == 0:
+            if ping_result:
                 break
-        cmd_result.assert_exit_code(
-            message=f"fail to ping {dest_ip} from {source_node.name} to "
-            f"{dest_node.name} after retry {max_retry_times}"
-        )
+        assert ping_result, f"fail to ping {dest_ip} from {source_node.name} to "
+        f"{dest_node.name} after retry {max_retry_times}"
```

Comment:
```
        assert ping_result, (
            f"fail to ping {dest_ip} from {source_node.name} to "
            f"{dest_node.name} after retry {max_retry_times} times"
        )
```

Reply:
without parentheses, only this ```f"fail to ping {dest_ip} from {source_node.name} to "``` can be printed out

Reply:
Done.